src/tplr/comms.py

```diff
@@ -1450,6 +1493,82 @@ async def post_start_window(self, start_window: int):
             if os.path.exists(temp_file):
                 os.remove(temp_file)
 
+    async def get_peer_list(self) -> tuple[int, int] | None:
+        tplr.logger.info("Starting to look for a peer list on a validator bucket")
+        while True:
```

Comment:
I got stuck in this loop while testing. It seems like the validator never uploaded the first one.